pkg/reconciler/route/resources/certificate.go

```diff
@@ -34,7 +34,7 @@ import (
 // MakeCertificate creates a Certificate, inheriting the certClass
 // annotations from the owner, as well as the namespaces. If owner
 // does not have a certClass, use the provided `certClass` parameter.
```

Comment:
Add a docstring about the `domain` parameter, particularly since it differs from the `dnsName` parameter in a possibly-subtle way?

Reply:
(you might want to call this `baseDomain`?)